source/class/qx/tool/cli/commands/package/Publish.js

```diff
@@ -79,6 +79,12 @@ qx.Class.define("qx.tool.cli.commands.package.Publish", {
           },
 
           dryrun: {
+            type: "boolean",
+            alias: "d",
```

Comment:
I think this is a problem. There are two options with the same alias, 'd'. Since `dryrun` is deprecated, it needn't have an alias.

Reply:
@derrell Do I need  just remove an alias for the deprecated option?

Reply:
Yes, I believe that's the correct solution. Leave the 'd' alias on `--dry-run` but remove it from `--dryrun`. I do not think it's required that an option has an alias.